docs/data/README.mdx

```diff
@@ -2,10 +2,27 @@
 title: Data Overview
 ---
 
-The RPC, data indexers, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
+Stellar supports tools that facilitate communication between users and applications on the Stellar network. Users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
 
 This section will walk you through the differences between the various platforms and tools, what platform or tool is best for what use case, and then link to their various documentation locations.
 
+- **RPC** - live network gateway
+- **Horizon** - API for network state data
+- **Galexie** - exports raw ledger metadata files
+- **Hubble** - analytics database for network data
+
+| Features                | RPC | Horizon | Galexie | Hubble |
```

Comment:
I think it may actually be better if this table was flipped. That is, the rows were the products/tools and the columns were the "features". This would also allow us to add additional columns to add more clarify. For example, the TLDR of each service could actually now be in the table.

Reply:
I took a look at the flipped version of the table. I think I like how it currently is (columns == services) because
1. it looks more ergonomic listing new "features" as rows instead of making a wide table that may not fit on the window without horizontal scrolling
2. it may just be a personal preference but fewer columns and more rows ("skinny table") seems to "read easier" kinda like how it's harder to read long lines https://baymard.com/blog/line-length-readability